src/widgets/panel/panel.tsx

```diff
@@ -252,6 +252,7 @@ const panel: PanelWidget = widgetParams => {
 
     return {
       ...widget,
+      $$props: widgetOptions,
```

Comment:
Can you explain what it is? I can't find `$$props` being used elsewhere.

Reply:
leftover from the old option, to remove, thanks!